<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新新魔塔存档编辑器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>82</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="import_button">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="destroy_button">
       <property name="text">
        <string>清除墙体</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replace_button">
       <property name="text">
        <string>替换图块</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_button">
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="about_button">
       <property name="text">
        <string>关于</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_bg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>416</width>
      <height>416</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="buddy">
     <cstring>label_bg</cstring>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
